test/cluster/cloud_cluster.bats

```diff
@@ -0,0 +1,104 @@
+# Note: Intended to be run as "make cloud-cluster-tests"
+#       Makefile target installs & checks all necessary tooling
+#       Extra tools that are not covered in Makefile target needs to be added in verify_prerequisites()
+
+load helpers_cloud
+load helpers_wait
+
+function setup_cloud_services() {
+  awslocal s3 --region "us-east-2" mb s3://zot-storage-test
+  awslocal dynamodb --region "us-east-2" create-table --table-name "BlobTable" --attribute-definitions AttributeName=Digest,AttributeType=S --key-schema AttributeName=Digest,KeyType=HASH --provisioned-throughput ReadCapacityUnits=10,WriteCapacityUnits=5
+}
+
+function launch_zot_server() {
```

Comment:
TODO: check if this function can be moved out to a shared bash script and imported as a module